app/bridge/client.js

```diff
@@ -33,6 +39,19 @@ function createNewClient(name, version) {
     smuggler.close();
   }
 
+  function unregister(client, uuid) {
+    var kRegistrationChannelName = 'smuggler';
+    var smuggler = new BroadcastChannel(kRegistrationChannelName);
+    smuggler.postMessage({
+      name: 'unregister',
+      type: 'client',
+      contract: client.name,
+      version: client.version || '',
```

Comment:
Why |version || ''| and not only version ?

Also, this code and the code of |register| are pretty closed. Can you create a method that takes the |name| as a parameter to factorize a bit the code.
Maybe |postCommand| or something similar.